.env.example

```diff
@@ -1 +1,5 @@
-GITHUB_PERSONAL_ACCESS_TOKEN=
+APP_ID=123456
```

Comment:
Can we document where we would find these env vars?

Reply:
I can update the README to include this information.

Reply:
Do you think it would be worth including how to install the app too? Presumably there is some Github documentation we can point to